src/Chapter2.hs

```diff
@@ -718,7 +727,7 @@ value of the element itself
 🕯 HINT: Use combination of 'map' and 'replicate'
 -}
 smartReplicate :: [Int] -> [Int]
-smartReplicate l = error "smartReplicate: Not implemented!"
+smartReplicate = concatMap (\x -> replicate x x)
```

Comment:
Very nice implementation with standard function `concatMap` and eta-reduction! :panda_face: